template/sample/template.gen_test.go

```diff
@@ -649,13 +652,37 @@ func (e *fakeExpr) Eval(mapExpression string, attrs attribute.Bag) (interface{},
 	if strings.HasSuffix(expr2, "timestamp") {
 		return time.Date(2017, time.January, 01, 0, 0, 0, 0, time.UTC), nil
 	}
-	ev, _ := expr.NewCEXLEvaluator(expr.DefaultCacheSize)
+	ev, _ := evaluator.NewILEvaluator(1024, 1024)
+	ev.ChangeVocabulary(descriptor.NewFinder(&baseConfig))
 	return ev.Eval(expr2, attrs)
 }
 
+var baseConfig = istio_mixer_v1_config.GlobalConfig{
+	Manifests: []*istio_mixer_v1_config.AttributeManifest{
+		{
+			Attributes: map[string]*istio_mixer_v1_config.AttributeManifest_AttributeInfo{
+				"str.absent": {
```

Comment:
Minor nit: consider using a not-dotted attribute name. If we ever change the expression semantics to support structured types, this might become an issue.